src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java

```diff
@@ -2,8 +2,8 @@
 
 public class IntLinkedList {
 
-	Node last;
-	int i;
+	private Node last;
+	private int i;
 
```

Comment:
value returned by top() is constant and should be extracted.

Reply:
Done